.travis.yml

```diff
@@ -7,13 +7,17 @@ python:
     - 3.6
     - nightly
 
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
 
 install:
     - pip install coveralls pytest==3.6 pytest-cov pycodestyle nibabel
 
 script:
-    - pycodestyle sam/imageutils.py tests/*.py
-    - pytest --cov=sam ./
+    - python -m pytest
```

Comment:
I know there is better that the way i used, but coveralls no longer works. Also, is there a reason for removing pycodestyle?

Reply:
The reason is that pycodestyle failed I don't know why so I removed it (it was just telling me that it failed with code 1 without other explanations). I also removed --cov because I did not know what it was and I just wanted to make it work. But we can add --cov back I just did not remember to do it.